src/SALib/analyze/delta.py

```diff
@@ -134,20 +136,22 @@ def sobol_first(Y, X, m):
     xr = rankdata(X, method='ordinal')
     Vi = 0
     N = len(Y)
+    Y_mean = Y.mean()
     for j in range(len(m) - 1):
         ix = np.where((xr > m[j]) & (xr <= m[j + 1]))[0]
         nm = len(ix)
-        Vi += (nm / N) * (Y[ix].mean() - Y.mean()) ** 2
+        Vi += (nm / N) * (Y[ix].mean() - Y_mean)**2

```

Comment:
Spaces around operators (PEP8)

Reply:
Sorry Will, I disagree with this interpretation.

PEP8 does not enforce spacing around the power operator (see [here](https://wiki.fysik.dtu.dk/ase/development/python_codingstandard.html#run-autopep8-py-on-your-code) for one example)

You wouldn't write `Y = x^2` as `Y = x 2` 
(i.e. you wouldn't put a space between the coefficient and its power)

Reply:
How about this as a compromise?

`Vi += (nm / N) * ((Y[ix].mean() - Y_mean)**2)`

Suggested in:

https://github.com/PyCQA/pycodestyle/issues/292#issuecomment-44591870

Reply:
Your compromise looks good. Thanks for looking into this!

Reply:
Your compromise looks good. Thanks for looking into this!